### DISCUSSION TITLE
mAkE cOaLtOn CaSe SeNsEtIve

### DISCUSSION THREAD

**Discussion by eliaslfox** [coalton-lang/coalton#156] [Ideas and Proposals]

**Comment by stylewarning** [comment]
Pro: It’s idiomatic in non-Lisp languages, and arguably objectively looks nicer.

Con: It means calling Coalton from regular Common Lisp requires `|EscapingEverything|`.

Con: It means the `LISP` operator would need its contents `WRITTEN IN ALL CAPS`.

Con: You’d have to have both `in-package` and `named-readtables:in-readtable` forms every time you wanted to use Coalton.

My sense is that making Coalton have separate _reader_ (as opposed to S-expression) syntax than Lisp will cause more pain than it’s worth. I know, I know, our eyes are so used to reading Standard ML, OCaml, Haskell, Rust, Elm, etc. But maybe this is a compromise we just have to make.
